scipy/stats/tests/test_continuous_basic.py

```diff
@@ -50,20 +50,51 @@
 # distslow are sorted by speed (very slow to slow)
 
 # skip check_fit_args (test is slow)
-skip_fit_test = ['exponpow', 'exponweib', 'gausshyper', 'genexpon',
-                 'halfgennorm', 'gompertz', 'johnsonsb', 'johnsonsu',
-                 'kappa4', 'ksone', 'kstwo', 'kstwobign', 'mielke', 'ncf', 'nct',
-                 'powerlognorm', 'powernorm', 'recipinvgauss', 'trapz',
-                 'vonmises', 'vonmises_line',
-                 'levy_stable', 'rv_histogram_instance']
+skip_fit_test_mle = ['exponpow', 'exponweib', 'gausshyper', 'genexpon',
+                     'halfgennorm', 'gompertz', 'johnsonsb', 'johnsonsu',
+                     'kappa4', 'ksone', 'kstwo', 'kstwobign', 'mielke', 'ncf',
+                     'nct', 'powerlognorm', 'powernorm', 'recipinvgauss',
+                     'trapz', 'vonmises', 'vonmises_line', 'levy_stable',
+                     'rv_histogram_instance']
+# these were really slow in `test_fit`.py.
+# note that this list is used to skip both fit_test and fit_fix tests
+slow_fit_test_mm = ['argus', 'exponpow', 'exponweib', 'gausshyper', 'genexpon',
+                    'genhalflogistic', 'halfgennorm', 'gompertz', 'johnsonsb',
+                    'kappa4', 'kstwobign', 'recipinvgauss', 'skewnorm',
+                    'trapz', 'truncexpon', 'vonmises', 'vonmises_line']
+# pearson3 fails due to something weird
+# the first list fails due to non-finite distribution moments encountered
```

Comment:
put this comment up on line 53 above `skip_fit_test_mle`

Reply:
I'm not sure I understand. It's not about `skip_fit_test_mle`; it's about `fail_fit_test_mm`, so it is located above `fail_fit_test_mm`.

Reply:
>Do you mean you would prefer to see all comments about all the separate lists in one block above the lists rather than above the list they correspond with?

I think that would be clearer when looking over the branching logic, especially if someone is unfamiliar.

Reply:
Mind if I get a third opinion?
@WarrenWeckesser thoughts?

Reply:
This looks OK to me. (Sorry for the nine month(!) delay--I lost track of the ping.)  @rlucas7, there have been updates since last April; do you still have questions about the comments for these lists of skipped distributions?